VMs/metrics/3_complete_baseimage.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+# Copyright (c) 2018 Intel Corporation
+# 
+# SPDX-License-Identifier: Apache-2.0
+
+source global_env.sh
+
+die() {
+	echo "Fatal: $*" >&2
+	exit 1
+}
+
+error() {
+	echo "Error: $*" >&2
+}
+
+message() {
+	echo "Message: $*"
+}
+
+if [ $# != 1 ]; then
+	error "Missing first argument"
+	message "Usage: $0 distro_config_dirname"
+	die
+fi
+
+CONFIG_DIR=$1
+
+if [ ! -f ${CONFIG_DIR}/${ENV_FILE} ]; then
+	die "Env file ${CONFIG_DIR}/${ENV_FILE} missing"
+fi
+
+source ${CONFIG_DIR}/${ENV_FILE}
+
+# We could be smarter and check if it is already running...
+
+VM_IP=$(virsh domifaddr ${VM_NAME} | tail -2 | head -1 | awk '{print $4}')
+VM_IP=${VM_IP%%/*}
+echo "IP of ${VM_NAME} is ${VM_IP}"
+
+message "Checking VM uname"
+VM_UNAME=$(ssh -F /dev/null -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -o IdentitiesOnly=yes -i ${PRIVKEYNAME} ${VM_USERNAME}@${VM_IP} uname -n)
```

Comment:
These `ssh`/`scp` options are used in a few places so I'd put them in a variable and reference that to reduce line length and make it easier to maintain those options.